ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -135,7 +135,8 @@
     # Note - had to have these set to `install-mock` to progress role
     # installation before changing config to restored DB from other system.
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.3.1
+      omero_server_release: 5.3.2
+      omero_server_upgrade: True
```

Comment:
Yes, I don't think a different version would be installed but I could imagine both your server restarting when you don't want it to or omego attempting other _stuff_.